tests/clients/cli/wrapper_collection.go

```diff
@@ -348,10 +348,24 @@ func (c *Collection) CreateIndex(
 	}
 
 	fields := make([]string, len(indexDesc.Fields))
+	orders := make([]bool, len(indexDesc.Fields))
+
 	for i := range indexDesc.Fields {
 		fields[i] = indexDesc.Fields[i].Name
+		orders[i] = indexDesc.Fields[i].Descending
+	}
+
+	orderedFields := make([]string, len(fields))
+
+	for i := range fields {
+		if orders[i] {
+			orderedFields[i] = fields[i] + ":DESC"
+		} else {
+			orderedFields[i] = fields[i] + ":ASC"
```

Comment:
thought: I wonder if we should skip ":ASC" altogether, otherwise it looks like the default value won't be tested.

Reply:
It's an interesting thought... By including ":ASC" we test two out of three paths (":DESC" and ":ASC") But if we don't include it, we still only test two out of three paths (":DESC" and unspecified.)

Should we change something?

Reply:
If you change `testultils.IndexedField.Descending` to `immutable.Option[bool]` you should be able to handle the default case, but not the error/len>2 case.